src/SubdomainNavBar/SubdomainNavBar.test.tsx

```diff
@@ -74,4 +80,19 @@ describe('SubdomainNavBar', () => {
 
     expect(searchResultsDialog).toBeInTheDocument()
   })
+
```

Comment:
Should isCentered be tested as well?

Reply:
Could do, but it's a visual only change and covered by the snapshots.. in other test cases I've added a lookup for the corresponding CSS class, but I'm not 100% convinced by this.

Reply:
Actually, changed my mind 😅 .. I have added coverage now because there's a default prop value so it's not just visual. See this commit https://github.com/primer/brand/pull/89/commits/ad5eabe800dd90d4c306678a8049f2702ce6d1e6

Reply:
Perfect!